### ISSUE TITLE
Absolutely unethical and against the open web

### ISSUE THREAD

**Issue by joshuamauldin** [RupertBenWiser/Web-Environment-Integrity#134]
I won’t break new ground here but it is impossible to see how this isn’t DRM for browsers that forces more ads and tracking on people. There is no benefit to website visitors and users, only to advertisers. 

The ethics underlying this are, at best, highly suspect. The framing is entirely disingenuous.

**Comment by rubyFeedback** [comment]
Agreed. Google has to respond with a "no-nonsense" statement about trying to take away user's freedom here.

This is not looking good. We are now miles away from a free world wide web.

(Actually W3C already pushed the envelope some years ago when Tim claimed that DRM is necessary, and now Google wings it by forcing it onto the users rendering them without control. That is making the browser itself a weapon for forcing unwanted content onto the user. It's now like a virus.)
